x-pack/plugins/data_enhanced/public/search/sessions_mgmt/lib/get_columns.tsx

```diff
@@ -99,8 +104,10 @@ export const getColumns = (
         );
         return (
           <RedirectAppLinks application={core.application}>
+            {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
             <EuiLink
-              href={isRestorable ? restoreUrl : reloadUrl}
+              href={href}
```

Comment:
Can't this cause some weird issues in different browser?
Maybe it's better turn this into a button and handle the report + navigation onClick?

Reply:
It turns out this is used a lot in our code base for these types of use cases... There are a few discussions in Slack about this if you search for `eslint-disable-next-line @elastic/eui/href-or-on-click` (e.g. https://elastic.slack.com/archives/C7QC1JV6F/p1584564657433200)

Reply:
I had changed it to use `navigateToUrl` but it messed up the tests... So I'm reverting it